overlord/patch/patch1_test.go

```diff
@@ -104,7 +105,9 @@ func (s *patch1Suite) TestPatch1(c *C) {
 	st, err := state.ReadState(nil, r)
 	c.Assert(err, IsNil)
 
-	err = patch.ApplyOne(patch.Patch1, st, 1)
+	// go from patch-level 0 to patch-level 1
+	patch.Level = 1
```

Comment:
We really want MockLevel as suggested, instead of directly fiddling with patch.Level. This would create problems elsewhere for lack of restore otherwise.